### ISSUE TITLE
Saved figure backgrounds are always transparent

### ISSUE THREAD

**Issue by slopedog** [matplotlib/ipympl#234]
## Describe the issue

I'm not positive that this is ipympl and not matplotlib, but when I click the button to save the plot, the image always has a transparent background. Using `fig.patch.set_facecolor('#111111')` doesn't seem to help.

![image](https://user-images.githubusercontent.com/8258310/84513842-0d844480-ac98-11ea-883b-c31592154597.png)

## Versions

```
 3.6.8 (default, Apr 25 2019, 21:02:35) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]
ipympl version: 0.5.6
jupyter core     : 4.6.3
jupyter-notebook : 6.0.3
qtconsole        : 4.7.4
ipython          : 7.15.0
ipykernel        : 5.3.0
jupyter client   : 6.1.3
jupyter lab      : 2.1.4
nbconvert        : 5.6.1
ipywidgets       : 7.5.1
nbformat         : 5.0.7
traitlets        : 4.3.3
Known nbextensions:
  config dir: /home/jah/env/etc/jupyter/nbconfig
    notebook section
      jupyter-matplotlib/extension  enabled 
      - Validating: OK
      jupyter-js-widgets/extension  enabled 
      - Validating: OK
JupyterLab v2.1.4
Known labextensions:
   app dir: /home/jah/env/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        jupyter-matplotlib v0.7.2  enabled  OK

```

**Comment by ianhi** [comment]
I think this is the correct behavior, in the rcparams there is a separate setting for the facecolor of savefig. See https://stackoverflow.com/a/4805178/835607

**Comment by slopedog** [comment]
That didn't seem to do the trick.
![image](https://user-images.githubusercontent.com/8258310/88406445-b301fc00-cd9e-11ea-86e4-bddbf8a52614.png)

**Comment by ianhi** [comment]
Maybe this is a matplotlib issue? I am using matplotlib version 3.2.1  and get this:
![image](https://user-images.githubusercontent.com/10111092/88426747-b5288280-cdbf-11ea-8e23-b91dd341d341.png)

**Comment by ianhi** [comment]
another check to isolate the cause is to try this with `%matplotlib inline` to see if this is due to ipympl or something else

**Comment by slopedog** [comment]
Yes - there is some version thing going on! I just tried with 3.2.1, and I got your backgrounds using your code. But using the save button next to the plot (which is what I was doing above) gives me a blue saved figure. So the button does something different from `savefig`. In the older matplotlib, both methods yield a transparent background.

So it appears that this is a limitation of older matplotlib, and also the button in the widget behaves differently from `savefig`.

**Comment by ianhi** [comment]
oooh I see, I somehow missed that you using the button to save the plot. Indeed if use matplolibt from the terminal and try the save button it saves with a red background: 
![image](https://user-images.githubusercontent.com/10111092/88428054-2a955280-cdc2-11ea-8b3e-ff062628e83d.png)

I think that the underlying issue here is the same as https://github.com/matplotlib/ipympl/issues/138

**Comment by slopedog** [comment]
Great - thanks for tracking this down!

**Comment by ianhi** [comment]
this should now work in the just released on pypi (soon on conda-forge) version 0.10.0
